doc/readme/INSTALL.md

```diff
@@ -130,6 +130,7 @@ In case none of the binary packages listed on our website work for your system,
 
 ## Requirements <a name="manual-requirements"></a>
 The standard GNU/Linux tools and Python are minimal requirements to compile Shogun.
+For the convenience of developers, the `BUILD_META_EXAMPLES` is initially `ON`, so we need to have `python-ply` and `ctags` be installed, see [generating meta examples](#manual-examples) below.
```

Comment:
By default, building the meta examples is enabled, requiring `python-ply` and `ctags`. Disable using `-DBUILD_META_EXAMPLES=Off` if those requirements are a problem, also see [generating meta examples](#manual-examples).

Reply:
Thanks! Your expression is more precise and accurate.